### DISCUSSION TITLE
Display non-Markdown files.

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [damionx7/Zettel-Notes-Documentation#70] [Feature Requests]
As https://github.com/damionx7/Zettel-Notes-Documentation/issues/45#issue-2091842526 states:

<blockQuote>

I'd like to be able to open the synchronized non-Markdown files in the native viewers I have assigned to them. Even if they shouldn't be displayed in the application GUI, at least providing access via a `com.android.documentsui` portal is enough. However, I'd like to be able to see them via the application GUI too.

</blockQuote>

**Comment by aamott** [comment]
Definitely agreed. I'm using this as a full notebook between my phone and PC and code editing is a big thing for me. Images and documentation are big too, and considering doing some voice journaling again.

**Comment by damionx7** [comment]
[Sam]: Add an option to list all files in a folder in the notes view, not just the markdown / text files. This would essentially act as a browser where non-note files are opened in the default external application.
Right now, you can access these files via the gallery, but that does not keep them in their hierarchical structure or in the context of the notes around them.

**Comment by damionx7** [comment]
I have added this feature. It will be live in the next update.

For more details, check out this telegram annoucement https://t.me/zettelnotes/351.
